client/src/pages/PaymentPortal.js

```diff
@@ -28,20 +32,31 @@ class PaymentPortal extends Component {
   componentDidMount(){
     //TODO: Figure out how it is being passed later : this.props.houseHoldID
     //TODO: Add logic for retrieving householdID
-    //TODO: Replact 5c8680ffad46ec4f26e7b46f with householdID from redux
-    let urlToFetch = API_STUDENT_PROD+ '5c8680ffad46ec4f26e7b46f';
+
+
+    const householdId = this.props.auth.household.householdId;
```

Comment:
I played with this for a bit and it's breaking because this `householdId` is the user-readable one, not the mongoose _id we need to pass to the API call. We should rename these in the front and backend to avoid further confusion.

Reply:
Is this good to merge in now? @Samyakkumar @spencerhyphengold?